coordinator/src/routes/admin.rs

```diff
@@ -623,6 +664,44 @@ pub async fn get_user_referral_status(
     Ok(Json(referral_status))
 }
 
+#[instrument(skip_all, err(Debug))]
+pub async fn post_funding_rates(
+    State(state): State<Arc<AppState>>,
+    Json(funding_rates): Json<FundingRates>,
+) -> Result<(), AppError> {
+    let mut conn = state
+        .pool
+        .get()
+        .map_err(|e| AppError::InternalServerError(format!("Could not get connection: {e:#}")))?;
+
+    for FundingRate {
+        rate,
+        start_date,
+        end_date,
+    } in funding_rates.0
+    {
+        let funding_rate = funding_fee::FundingRate::new(rate, start_date, end_date);
+
+        if let Err(e) = db::funding_rates::insert(&mut conn, &funding_rate) {
```

Comment:
Should we do this in a transaction on all inserts and return an error if one fails? Otherwise it is impossible for a caller to know which was inserted and which one failed without looking into the logs.